src/test/java/org/mockito/internal/verification/checkers/NumberOfInvocationsInOrderCheckerTest.java

```diff
@@ -27,6 +22,11 @@
 import org.mockito.invocation.Invocation;
 import org.mockitousage.IMethods;
 
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
```

Comment:
Same thing with imports as above.

Reply:
Here the imports also violates the described import layout